etcdserver/cluster_util.go

```diff
@@ -195,15 +194,15 @@ func isCompatibleWithVers(vers map[string]*version.Versions, local types.ID, min
 		}
 		clusterv, err := semver.NewVersion(v.Cluster)
 		if err != nil {
-			log.Printf("etcdserver: cannot understand the cluster version of member %s (%v)", id, err)
+			plog.Warningf("cannot understand the cluster version of member %s (%v)", id, err)
```

Comment:
why it is warningf here instead of errorf as this line?
`plog.Errorf("cannot understand the version of member %s (%v)", mid, err)`